test/utils/test-swagger-gen.ts

```diff
@@ -819,6 +819,16 @@ const testSwaggers: { [name: string]: SwaggerConfig } = {
     restLevelClient: true,
     licenseHeader: true,
     isTestPackage: true
+  },
+  textAnalytics: {

```

Comment:
let's remove the entry at line 698 because it is a duplicate and gets overwritten by the new test you add here.

Reply:
I renamed it, thanks for pointing this out :)

Reply:
I think you should delete the old entry all together, there is no need to generate the same swagger twice in our tests. Technically the two swaggers are not exactly the same but the one you're adding have everything in the old one and more so let's stick with that instead.

Reply:
That makes sense, @sarangan12 what are your thoughts on this?

Reply:
@JonathanCrd Please check if the old entry has this boolean parameter. If so, You can delete it.